docs/topics/mta-rn-new-features-6-2-0.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * docs/release_notes/master.adoc
+
+:_content-type: CONCEPT
+[id="rn-new-features-6-2-0_{context}"]
+= New features
+
+
+This section describes the new features of the Migration Toolkit for Applications (MTA) 6.2.0.
+
+.Integration with JIRA
+
+The integration of {ProductName} with Jira allows you to track and manage the whole migration process, and delegate implementation to external issue managers or task managers.
+
+//// 
+For more information, see xref:../../docs/topics/mta-web-config-jira-connection.adoc[Configuring Jira connection]
+////
+
+.Migration Waves management
+
+A migration wave is a small collection of workloads that deliver business value. {ProductShortName}'s _Migration Wave_ groups applications to be migrated on a specified schedule.
+
+In addition, a migration wave enables you to export a list of the wave's applications to the Jira issue management system. This automatically creates a separate Jira issue for each application of the migration wave for tracking.
+
+////
+For more information, see xref:../../docs/topics/mta-web-creating-migration-waves.adoc[Creating migration waves].
+////
+
+.OpenShift Monitoring integration
```

Comment:
Hi @anarnold97 Not sure but why `.OpenShift Monitoring integration` not been highlighted with blue marker ?

Reply:
It is an issue with GitHub not reading the above block comment properly. 

![image](https://github.com/windup/windup-documentation/assets/106804821/62543a44-01c8-4565-938c-27231a1abcb8)

Checked in AsciiDocLive and it reads correctly

Reply:
Oh Okay i see if it's working good will approve the PR then. Thanks!

Reply:
Here it is rendered in AsciiDocLive

![image](https://github.com/windup/windup-documentation/assets/106804821/4d3286ca-37c6-47cc-864c-2ff1e4c7733c)

It is so annoying as I spent ages trying to figure out what was wrong.LOL

Also remember when I come to finally publishthis doc,  the comments will be removed as the links should then be valid

Reply:
No worries! Thanks @anarnold97 for your effort !

Reply:
LGTM